package.json

```
@@ -17,6 +17,7 @@
     "axios": "^1.13.4",
```

Biome (2.3.14)

[error] The key axios was already declared.

This is where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)